release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -172,6 +172,21 @@ The {product-title} internal registry and image streams now support Open Contain
 [id="ocp-4-7-machine-api"]
 === Machine API
 
+[id="ocp-4-7-aws-tenancy-dedicated"]
+==== Machine sets running on AWS support Dedicated Instances
+Machine sets running on AWS now support Dedicated Instances. Configure Dedicated Instances by specifying a dedicated tenancy under the `providerSpec` field in the machine set YAML file. For more information, see
```

Comment:
For consistency, I'd add a new line between the title and description to match the other RNs.

I would also recommend commenting out the entire sentence starting with `For more information,...`. We'll get comments in the tracker of a typo here if we have a sentence without an end. :smile:

Reply:
@codyhoag Thank you! I always forget how to handle the "For more information ..." part with these pending release notes hah.